src/lib/components/Autocomplete/Autocomplete.svelte

```diff
@@ -56,7 +56,7 @@
 				{/if}
 			{:else}
 				{#each Object.entries(makeGroups(suggested)) as [group, items], i}
-					{#if group && items.length}
+					{#if group && group !== 'undefined' && items.length}
```

Comment:
@Valexr a string can be equal to `"undefined"` :fearful: ?

Reply:
Yep, it’s quickly fix for `suggested: { group: [ items ] }` object. Undefined generate automatically, if group name not provided, but it can be any other name🤷🏻‍♂️by default… & it’s + lines of code. Without this simple hack - needed more conditions in template & groups generator function (++lines of code 🤓)